neurodamus/utils/version.py

```diff
@@ -0,0 +1,62 @@
+"""git_version.py
+
+Provides utilities for handling Git-style version strings (e.g., '9.0a-1485-g0d990513b').
+
+Includes:
+- GitVersion: class representing a version with base, commit count, and hash,
+  supporting comparison operators.
+- check_program_version: function to verify a program meets a minimum Git-version requirement.
+
+Intended for use with programs that use Git-describe-style versioning, such as NEURON.
+"""
+
+import re
+from functools import total_ordering
+
+from packaging import version
+
+
+def check_environment():
+    """Ensure programs in the environment (like neuron) are compatible with Neurodamus."""
+    import neuron
+
+    # Neurodamus requires NEURON >= 9.0a-1485-g0d990513b because
+    # the layout of `report.conf` changed in this commit.
+    req_version = GitVersion("9.0a-1485-g0d990513b")
```

Comment:
After NEURON 9 is released, the versioning is 9.0.0-xxx-xxxx. Is "9.0.0" > "9.0a" in the comparison?